src/cmfmcsop.cpp

```diff
@@ -412,7 +412,7 @@ bool CcmfmacsoperaPlayer::advanceRow()
 				currentOrderIndex++;
 
 				// check bounds
-				if (currentOrderIndex < 0 || currentOrderIndex >= sizeof(patternOrder) / sizeof(patternOrder[0]))
+				if (currentOrderIndex >= (sizeof(patternOrder) / sizeof(patternOrder[0])))
```

Comment:
When currentOrderIndex is unsigned and set to  ~0 when not set, test can be simplified

When currentOrderIndex is unsigned, it matches sizeof() being unsigned